admin_ui/src/components/EditRowForm.vue

```diff
@@ -1,51 +1,37 @@
 <template>
     <div v-if="schema">
         <div class="header">
-            <h1>Edit {{ tableName | readable }}</h1>
-
-            <p>
-                <a
-                    class="subtle"
-                    href="#"
-                    v-on:click.prevent="showDropdown = !showDropdown"
-                >
-                    <font-awesome-icon icon="ellipsis-v" />
-                    <DropDownMenu v-if="showDropdown">
-                        <li>
-                            <DeleteButton
-                                :includeTitle="true"
-                                class="subtle"
-                                v-on:triggered="deleteRow"
-                            />
-                        </li>
-                    </DropDownMenu>
-                </a>
-            </p>
```

Comment:
Did we lose this because it's possible to delete from the row list page?

Reply:
We already have possibility to delete single row from the row list page and I remove delete from edit page because when delete row I get `404` for deleted row id (`Vuex` doesn't update state of all rows). It was easier to me to remove option to delete row from edit.

Reply:
This feature returns with commit [7364542](https://github.com/piccolo-orm/piccolo_admin/pull/89/commits/736454244de6cfc006d16d766ab016a61b9613f2).